### PR TITLE
Add convenience keys for gateway_intents config

### PR DESCRIPTION
**Pull request by BrandtHill**
Yes

### CODE REVIEW

File: lib/nostrum/shard/intents.ex [hunk 1]
```diff
@@ -26,13 +31,18 @@ defmodule Nostrum.Shard.Intents do
...
   @spec get_enabled_intents :: integer()
   def get_enabled_intents do
-    enabled_intents = Application.get_env(:nostrum, :gateway_intents)
-
     # If no intents are passed in config, default to all being enabled.
-    if enabled_intents do
-      get_intent_value(enabled_intents)
-    else
-      get_intent_value(Keyword.keys(intent_values()))
+    enabled_intents = Application.get_env(:nostrum, :gateway_intents, :all)
```
Comment (Kraigie) [inline lib/nostrum/shard/intents.ex:35]: How do we feel about making the default be `:nonprivileged` so people who don't know about this setting still will have a working bot?

Comment (BrandtHill) [reply lib/nostrum/shard/intents.ex:35]: I would prefer that as it's minimally invasive for existing bots' configs. I just kept it as all to preserve the original author's intention.

Comment (Kraigie) [reply lib/nostrum/shard/intents.ex:35]: 👍 lets change it then

Comment (BrandtHill) [reply lib/nostrum/shard/intents.ex]: oui

**Comment by Kraigie** [comment]
Thanks!

**Pull request merged**